### DISCUSSION TITLE
Ignore Images for set period of time

### DISCUSSION THREAD

**Discussion by wraggzeh** [neilenns/node-deepstackai-trigger#408] [Q&A] [answered]
Would it be possible to implement some way of ignoring images for a set period of time? Something like messaging the Telegram Chat where I receive the alerts to ignore for 30 mins, the application sees this, then sleeps for that period of time?

E.g I have a Camera on my Driveway and I want to wash my car, but don't want it to trigger alerts via Telegram/Home Assistant etc all the time when I'm moving around.

**Comment by neilenns** [comment, marked as answer]
For this type of complex automation/rules I recommend using another system in conjunction with this project. Personally I use HomeAssistant and NodeRed for this kind of thing:

1. This project simply does AI detection then sends an MQTT message out alerting that something was detected
2. NodeRed receives the MQTT message and then decides whether it's appropriate to send a web request to BlueIris to start recording

This type of setup works well, and allows you to do things like have a button in your mobile Home Assistant app to disable notifications/recording/whatever.
